test/README.md

```diff
@@ -17,4 +17,4 @@ Example-driven test consists of two parts:
 
 ## Contributing
 
-Please see the [Contribution guidelines](../.github/CONTRIBUTING.md) for the instructions on how to run and add new tests. Thank you.
+Please see the [Contribution guidelines](/CONTRIBUTING.md) for the instructions on how to run and add new tests. Thank you.
```

Comment:
I think this path should begin with `./`, shouldn't it?

Reply:
@kettanaito I don't think so. We actually want to use the absolute path. If we want to use a relative path, we have to write like this
```md
(../CONTRIBUTING.md)
```
In any `.md` file in any folder we can use (/CONTRIBUTING.md)